### ISSUE TITLE
Incompatible with WP-CLI 2.8.0

### ISSUE THREAD

**Issue by johnbillion** [szepeviktor/phpstan-wordpress#188]
WP-CLI 2.8.0 was released yesterday. One of the larger changes was to upgrade to Requests v2, which means its autoloader now looks for and loads classes in the `\WpOrg\Requests` namespace.

The `szepeviktor/phpstan-wordpress` package depends on the `php-stubs/wordpress-stubs` package and [loads it when running PHPStan](https://github.com/szepeviktor/phpstan-wordpress/blob/6ee8aadb1cec8e44ff8ce9338f2a744c08ccd945/extension.neon#L116). The `php-stubs/wordpress-stubs` package includes the `\WpOrg\Requests` classes in its stubs.

Running the PHPStan analysis on a project that depends on both `szepeviktor/phpstan-wordpress` and `wp-cli/wp-cli`results in a redeclaration error. I've not dug into the details but it appears the stubs definitions load first, then something inside Requests still causes it to attempt to redeclare `\WpOrg\Requests\Autoload`. I will try to find some time to look into why that's happening.

```
phpstan analyze --memory-limit=1024M
Note: Using configuration file phpstan.neon.dist.
Fatal error: Cannot declare class WpOrg\Requests\Autoload, because the name is already in use in vendor/php-stubs/wordpress-stubs/wordpress-stubs.php on line 18381
```

**Comment by chesio** [comment]
Perhaps related: https://github.com/wp-cli/wp-cli/issues/5795

**Comment by johnbillion** [comment]
Some more information: The project does not depend directly on `wp-cli/wp-cli`, it depends on `lucatume/wp-browser` for automated testing, and that package depends on `wp-cli/wp-cli`.

**Comment by szepeviktor** [comment]
Please use `php-stubs/wp-cli-stubs` for static analysis.

**Comment by justlevine** [comment]
@chesio not "perhaps related", but what prompted me to open that issue in the first place 😇

Temporary workaround is to set `wp-cli/wp-cli` to `~2.7.x` locally, to stop other deps from leaking the higher version.

**Comment by szepeviktor** [comment]
I am sure there is a WP-CLI expert around here: https://github.com/php-stubs/wp-cli-stubs/issues/13
Please help!

**Comment by johnbillion** [comment]
Reading through the issue above, I think this is a problem in WP-CLI. It's trying to redeclare a `\WpOrg\Requests` class after it's already defined by WordPress core (or in this case, by the stubs).

**Comment by johnbillion** [comment]
This has been fixed in WP-CLI 2.8.1
